packages/plugin-autocapture-browser/src/autocapture-plugin.ts

```diff
@@ -198,6 +199,7 @@ export const autocapturePlugin = (options: AutocaptureOptions = {}): BrowserEnri
       [constants.AMPLITUDE_EVENT_PROP_PAGE_TITLE]: (typeof document !== 'undefined' && document.title) || '',
       [constants.AMPLITUDE_EVENT_PROP_VIEWPORT_HEIGHT]: window.innerHeight,
       [constants.AMPLITUDE_EVENT_PROP_VIEWPORT_WIDTH]: window.innerWidth,
+      [constants.AMPLITUDE_EVENT_PROP_AUTOCAPTURE_VERSION]: VERSION,
```

Comment:
Curious what is this value used for? FYI, some plugin replaces the `library` field to include the plugin/version information, https://github.com/amplitude/Amplitude-TypeScript/blob/537585f50779c8d4f465f21f99aba389a0d5d08e/packages/plugin-ga-events-forwarder-browser/src/constants.ts#L26.